From: Didier Raboud <odyx@debian.org>
Date: Sun, 4 Oct 2020 16:29:03 +0200
Subject: models.dat: 3.20.6+ regression: take 3.20.5's "plugin=1" for all
 models with 1 in least significant bit of plugin-reason (="needs plugin for
 printing")

---
 data/models/models.dat | 68 +++++++++++++++++++++++++-------------------------
 1 file changed, 34 insertions(+), 34 deletions(-)

diff --git a/data/models/models.dat b/data/models/models.dat
index 8d4f79d..be38c94 100644
--- a/data/models/models.dat
+++ b/data/models/models.dat
@@ -17763,8 +17763,8 @@ model1=HP LaserJet P1009 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
-plugin-reason=0
+plugin=1
+plugin-reason=1
 power-settings=0
 pq-diag-type=0
 r-type=0
@@ -18406,7 +18406,7 @@ model1=HP LaserJet Pro CP1025 Color Printer Series
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19042,7 +19042,7 @@ model1=HP LaserJet Professional P 1102w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19085,7 +19085,7 @@ model1=HP Laserjet Professional P1106w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19128,7 +19128,7 @@ model1=HP Laserjet Professional P1106 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19171,7 +19171,7 @@ model1=HP LaserJet Professional P1107 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19214,7 +19214,7 @@ model1=HP LaserJet Professional P1107w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19257,7 +19257,7 @@ model1=HP Laserjet Professional P1108w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19300,7 +19300,7 @@ model1=HP Laserjet Professional P1108 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19343,7 +19343,7 @@ model1=HP LaserJet Professional P1109w Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19386,7 +19386,7 @@ model1=HP LaserJet Professional P1109 Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -19736,7 +19736,7 @@ model2=HP LaserJet Professional M1132s Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19779,7 +19779,7 @@ model1=HP Laserjet Professional M1136 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19822,7 +19822,7 @@ model1=HP LaserJet Professional M1137 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19865,7 +19865,7 @@ model1=HP LaserJet Professional M1138 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -19908,7 +19908,7 @@ model1=HP LaserJet Professional M1139 Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20500,7 +20500,7 @@ model1=HP LaserJet Professional M1212nf Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20543,7 +20543,7 @@ model1=HP LaserJet Professional M1213nf Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20586,7 +20586,7 @@ model1=HP LaserJet Professional M1214nfh Multifunction Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20728,7 +20728,7 @@ model1=HP LaserJet Professional M1216nfh MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20771,7 +20771,7 @@ model1=HP Color LaserJet CP1217 Printer
 monitor-type=0
 panel-check-type=1
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -20866,7 +20866,7 @@ model1=HP Laserjet M1210 MFP Series
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20956,7 +20956,7 @@ model1=HP HotSpot LaserJet Pro M1218nfs MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -20998,7 +20998,7 @@ linefeed-cal-type=0
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -21040,7 +21040,7 @@ linefeed-cal-type=0
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -21083,7 +21083,7 @@ model1=HP LaserJet Professional M1219nf MFP
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=65
 power-settings=0
 pq-diag-type=0
@@ -24162,7 +24162,7 @@ model1=HP LaserJet Professional P1567
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24205,7 +24205,7 @@ model1=HP LaserJet Professional P1568
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24248,7 +24248,7 @@ model1=HP LaserJet Professional P1569
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24684,7 +24684,7 @@ model1=HP LaserJet Professional P1606dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24727,7 +24727,7 @@ model1=HP LaserJet Professional P1607dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24770,7 +24770,7 @@ model1=HP LaserJet Professional P1608dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
@@ -24813,7 +24813,7 @@ model1=HP LaserJet Professional P1609dn Printer
 monitor-type=0
 panel-check-type=0
 pcard-type=0
-plugin=0
+plugin=1
 plugin-reason=1
 power-settings=0
 pq-diag-type=0
